client/pages/quizzes.js

```diff
@@ -108,13 +108,13 @@ export default function Quizzes() {
         chosenTopics={chosenTopics}
         setChosenTopics={setChosenTopics}
       />
-      {
-        // Comment out until DB is ready
-        //  {!!error && <DisplayMessage message="Error loading quizzes" />}
-        // {!error && !data && <DisplayMessage message="... Loading Quizzes" />}
-        // change 'dummyData' to 'data' when DB is ready
+      {/* {
+        !!error && (<DisplayMessage message="Error loading quizzes" />)
       }
-      {!!dummyData && (
+      {
+        !error && !data && (<DisplayMessage message="... Loading Quizzes" />)
```

Comment:
For this I would probably create a variable like "loading" or "isLoading", so people don't have to read the message to get what it's doing 🙂

Reply:
@Soreg This was my setup for checking the client side fetch.

The SWR hook (recommended by Next for fetching on the client side) returns `data` and `error` and this is checking to see if there was an error or if data has not returned yet and returning the appropriate display message without having to create another piece of state to track it through or separate components for each message.

This could be updated to run through a `useEffect` hook and looped into a `loading` state, but the docs for SWR recommend this setup, also less code overhead and state to update as the SWR hook will update and re-render the component as needed.